packages/trip-form/src/MetroModeSelector/AdvancedModeSettingsButton.story.tsx

```diff
@@ -111,7 +120,8 @@ export default {
   argTypes: {
     fillModeIcons: { control: "boolean" },
     onSetModeSettingValue: { action: "set mode setting value" },
-    onToggleModeButton: { action: "toggle button" }
+    onToggleModeButton: { action: "toggle button" },
+    onAllSubmodesDisabled: { action: "all submodes disabled" }
```

Comment:
I don't see "all submodes disabled" appearing in the Storybook action pane, did I miss something?

Reply:
> I don't see "all submodes disabled" appearing in the Storybook action pane, did I miss something?

I pulled a previous commit, sorry, ignore that comment.